services/web-app/pages/contributing/schema.mdx

```diff
@@ -248,7 +278,7 @@ assets:
 | `noIndex`         | If set to `true`, the global catalogs will exclude the asset.                                                                                            | Optional | Boolean          | No          | `false`       | –                                                                                                                                                                                                                                                                                                                                          |
 | `docs`            | Contains information on where to find supporting documentation for the asset. See [asset docs](#asset-docs).                                             | Optional | Object           | Yes         | -             | –                                                                                                                                                                                                                                                                                                                                          |
 
-#### Asset type
+### Asset type
```

Comment:
can we move Asset type above Asset keys? I get that when defining the new object you will want to fill out the keys but as a casual reader it feels easier to start with 'type' to get a grasp on what assets can be.

Reply:
I think that would break our documentation pattern where we show all available keys, then below we describe each key in more detail when necessary.